sdk/communication/Azure.Communication.Sms/tests/samples/Sample1_SmsClient.cs

```diff
@@ -4,15 +4,113 @@
+
+            #region Snippet:SendSms
+            /// Send an sms
+            SmsSendResult sendSmsResult = smsClient.Send(
+                from: "<leased-phone-number>",
+                to: "<to-phone-number>",
+                message: "<message-to-send>");
+
+            Console.WriteLine("MessageId: " + sendSmsResult.MessageId);
+            #endregion Snippet:SendSms
+        }
+
+        [Test]
+        [SyncOnly]
+        public void SendSingleAndGroupSms()
+        {
+            SmsClient smsClient = CreateSmsClient();
+
+            #region Snippet:SendSms
+            /// Send an sms to a single recipient
+            SmsSendResult sendSmsResult = smsClient.Send(
+                from: "<leased-phone-number>",
+                to: "<to-phone-number>",
+                message: "<message-to-send>",
+                options: new SmsSendOptions // OPTIONAL
+                    {
+                    EnableDeliveryReport = true,
+                    Tag = "<custom-tags>",
+                });
+
+            Console.WriteLine("MessageId: " + sendSmsResult.MessageId);
+            #endregion Snippet:SendSms
+
+            #region Snippet:SendGroupSms
+            /// send an sms to multiple recipients
+            Pageable<SmsSendResult> results = smsClient.Send(
+                from: "<leased-phone-number>",
+                to: new[] { "<to-phone-number-1>", "<to-phone-number-2>", "<to-phone-number-3>" },
+                message: "<group-message-to-send>",
+                options: new SmsSendOptions // OPTIONAL
+                    {
+                    EnableDeliveryReport = true,
+                    Tag = "<custom-tags>",
+                });
+            foreach (SmsSendResult result in results)
+            {
+                Console.WriteLine($" MessageId: {result.MessageId} Sent to: {result.To}");
+            }
+            #endregion Snippet:SendGroupSms
+        }
+
+        [Test]
+        [AsyncOnly]
+        public async Task SendSingleAndGroupSmsAsync()
+        {
+            SmsClient smsClient = CreateSmsClient();
+
+            // SendSmsOptions is an optional field. It can be used
+            // to enable a delivery report to the Azure Event Grid
+            // and to add custom tags in the request
+            SmsSendOptions sendSmsOptions = new SmsSendOptions
+            {
+                EnableDeliveryReport = true,
+                Tag = "<custom-tags>",
+            };
+
+            #region Snippet:SendSms
+            /// send an sms to a single recipient asynchronously
+            SmsSendResult sendSmsResult = await smsClient.SendAsync(
+                from: "<leased-phone-number>",
+                to: "<to-phone-number>",
+                message: "<message-to-send>",
+                sendSmsOptions); //OPTIONAL
+            Console.WriteLine($" MessageId: {sendSmsResult.MessageId} Sent to: {sendSmsResult.To}");
+            #endregion Snippet:SendSms
+
+            #region Snippet:SendGroupSms
```

Comment:
We normally we go with this type of format to ensure we avoid duplication:

Snippet:Azure_Communication_SmsClient_Send_GroupAsync

`<namespace>_<client>_<method>_<extra description>`